editoast/src/core/conflict_detection.rs

```diff
@@ -78,16 +78,65 @@ pub struct ConflictDetectionResponse {
 #[derive(Debug, Clone, PartialEq, Deserialize, Serialize, ToSchema)]
 pub struct PacedTrainOccurrenceId {
     paced_train_id: i64,
-    index: i64,
+    index: i32,
 }
 
 #[derive(Debug, Clone, PartialEq, Deserialize, Serialize, ToSchema)]
```

Comment:
Are all these derives still necessary with the introduction of `ConflictResponse`?